test/common/fixtures/ingresses.networking.k8s.io.yaml

```diff
@@ -0,0 +1,8 @@
+kind: fixture
```

Comment:
perhaps a rebase needed given https://github.com/kubernetes-sigs/kubefed/pull/1521 is merged.

Reply:
:+1: that's why i blocked it, thanks for reminding me.

Reply:
Done.